packages/back-end/src/models/FactMetricModel.ts

```diff
@@ -183,8 +189,13 @@ export function toFactMetricApiInterface(
     ...otherFields
   } = omit(factMetric, ["organization"]);
 
+  const metricType = otherFields.metricType;
+  if (metricType === "quantile") {
+    throw new Error("Quantile metrics not avaiable yet");
+  }
```

Comment:
Doesn't seem like we should throw here.  If someone does happen to create a quantile metric in Mongo, this code will cause the `GET /fact-metrics` endpoint to fail, so they won't even be able to get a list of metrics from the API anymore.

Reply:
Ok! Seems reasonable enough to remove this for now. DB messers beware.

Reply:
I just set metricType to "mean" if it is "quantile" for now!